lib/phase/flatten.js

```diff
@@ -13,7 +13,7 @@ const LABEL_MATCH = 7;
  * @param  {Object}   newJob    Job to be merged into. For ex: shared settings
  * @param  {Object}   oldJob    Job to merge. For ex: individual job settings
  */
-function merge(newJob, oldJob) {
+function merge(newJob, oldJob, fromTemplate) {
```

Comment:
Yeah I wanted it to be `false` by default since this is also called somewhere else, and I don't want to mess merging shared into jobs: https://github.com/screwdriver-cd/config-parser/blob/master/lib/phase/flatten.js#L57